lib/module.js

```diff
@@ -14,3 +15,6 @@ export default async function nuxtRedirect(moduleOptions) {
 
   this.addServerMiddleware(middleware)
 }
+
+module.exports = redirectModule
```

Comment:
if it's ESM syntax, keep it ESM syntax.

Reply:
I want to do in the next PR the change to ESM

Reply:
` export default async function nuxtRedirect(moduleOptions) {`

It *was* already ESM

Reply:
If it is not ESM we open coverage for nuxt 1x

Reply:
In my opinion we should release a version for Nuxt 1x and later a major release only covering Nuxt 2x

Reply:
I agree as mentioned before.

Forgot that this plugin didn't include a notice in the README, so yeah, let's do it the way you suggested.